graylog2-server/src/main/java/org/graylog2/shared/security/RestPermissions.java

```diff
@@ -67,6 +67,7 @@
     public static final String INDICES_DELETE = "indices:delete";
     public static final String INDICES_FAILURES = "indices:failures";
     public static final String INDICES_READ = "indices:read";
+    public static final String INPUTS_CHANGESTATE = "inputs:changestate";
```

Comment:
Please add the new `INPUTS_CHANGESTATE` permission to the `PERMISSIONS` Set below.

Reply:
✅